packages/react/src/components/MultiSelect/MultiSelectPropTypes.ts

```diff
@@ -46,14 +46,14 @@ interface SharedOptions {
   locale: string;
 }
 
+interface CompareItems {
+  (itemA: string, itemB: string, options: SharedOptions): number;
```

Comment:
Are we losing some strictness here by moving the items from `ItemType` to `string`?

Reply:
Wasn't that the problem, that the items had type `ItemType` instead of `string`? Aren't the items going through `itemToString` before they go to the comparison function?